src/recipes.jl

```diff
@@ -192,6 +192,12 @@ end
 if VERSION >= v"1.6.0-DEV.890"   # https://github.com/JuliaLang/julia/pull/37320
     push!(stdlib_names, :Artifacts)
 end
+if VERSION >= v"1.6.0-DEV.938"   # https://github.com/JuliaLang/julia/pull/37340
+    push!(stdlib_names, :LibCURL_jll)
+    push!(stdlib_names, :LibCURL)
+    push!(stdlib_names, :MozillaCACerts_jll)
```

Comment:
I think these three are just temporary so might have to have an upper Julia version bound for these later on.